migrations/0046_db_optimizations.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX claim_drain_pending ON claim_drain (status, updated_at) WHERE status LIKE '%pending%';
```

Comment:
Will these create index statements block the table for writes, if so is that okay, is `CONCURRENTLY` an option